tests/Jobs/RequeuePendingBatchesJobTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\QsBatch;
+use App\Jobs\RequeuePendingBatchesJob;
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Contracts\Queue\Job;
+use Carbon\Carbon;
+
+class RequeuePendingBatchesJobTest extends TestCase
+{
+
+    use RefreshDatabase;
+
+    public function setUp(): void {
+        // Other tests leave dangling wikis around so we need to clean them up
+        parent::setUp();
+        QsBatch::query()->delete();
+    }
+
+    public function tearDown(): void {
+        QsBatch::query()->delete();
+        parent::tearDown();
+    }
+
+    public function testRequeue (): void {
```

Comment:
This could probably also test that errors are reported, but I have yet to find how to fake a global helper (i.e. there's no facade for `report` it seems)